mxcube3/routes/beamline.py

```diff
@@ -83,7 +83,7 @@ def set_func():
         args = request.get_json()
         adapter = app.mxcubecore.get_adapter(obj.lower())
         adapter.execute_command(cmd_name, args)
-        return make_response("{}", 200)
+        return "Command executed successfull"
```

Comment:
successfully

Reply:
Are there cases where make_response returns None? When? Why?

Reply:
seems so, all the calls in this method, no clue why

Reply:
hm, odd